util-logging/src/main/scala/com/twitter/logging/App.scala

```diff
@@ -58,14 +60,15 @@ trait Logging { self: App =>
     val level = Some(levelFlag())
     var handler =
       if (output == "/dev/stderr")
-        ConsoleHandler(level = level)
+        ConsoleHandler(level = level, formatter = defaultFormatter)
       else
         FileHandler(
           output,
           rollPolicyFlag(),
           appendFlag(),
           rotateCountFlag(),
-          level = level
+          level = level,
+          formatter = defaultFormatter
```

Comment:
mind keeping the ordering the same as the method's? in other words, just move formatter right before level.